docs/topics/gradle.md

```diff
@@ -722,6 +754,55 @@ See the [Gradle documentation](https://docs.gradle.org/current/userguide/configu
 to learn how to enable the configuration cache. After you enable this feature, the Kotlin Gradle plugin will automatically
 start using it.
 
+## Build reports
+
+> Build reports are [Experimental](components-stability.md). They may be dropped or changed at any time.
+> Opt-in is required (see details below). Use them only for evaluation purposes. We appreciate your feedback on them
+> in [YouTrack](https://youtrack.jetbrains.com/issues/KT).
+>
+{type="warning"}
+
+Starting with [Kotlin 1.7.0](whatsnew17.md#build-reports-for-kotlin-compiler-tasks), build reports for tracking compiler
```

Comment:
For me, this line is blog style.
I'd recommend to use something like "The build reports for ... is available for Kotlin 1.7.0"

WDYT?